### PR TITLE
BC buttons: Add sequential quote play toggle

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

**Comment by NoSloppy** [comment]
Actually, now I see there's something wrong here. Toggling back to NOT use SelectNext(), the quotes are not randomized again, but instead play the last selected every time.....hmmm.
What needs to make them random again?

File: props/saber_BC_buttons.h [hunk 1]
```diff
@@ -912,6 +923,14 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       if (fusor.angle1() >  M_PI / 3) {
         SaberBase::DoEffect(EFFECT_POWERSAVE, 0);
         return true;
+      } else if (fusor.angle1() < - M_PI / 4) {
+      // pointing down
+        sequential_quote_ = !sequential_quote_;
+        beeper.Beep(0.1, 1300);
```
Comment (profezzorn) [inline props/saber_BC_buttons.h:929]: Might want to make these beeps different when sequential is turned on vs. off.
Like maybe hi/lo/hi/lo is on and lo/hi/lo is off?
Or maybe use sound library?

Comment (NoSloppy) [reply props/saber_BC_buttons.h]: Good Idea, thanks.
Also updated instructions for this feature.

File: props/saber_BC_buttons.h [hunk 2]
```diff
@@ -912,6 +928,18 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
...
+      } else if (fusor.angle1() < - M_PI / 4) {
+      // pointing down
+        sequential_quote_ = !sequential_quote_;
+        if (sequential_quote_) {
+          beeper.Beep(0.1, 600);
+          beeper.Beep(0.1, 950);
+          beeper.Beep(0.1, 1300);
+          beeper.Beep(0.1, 600);
+        } else {
+          sound_library_.SayRandom();
```
Comment (profezzorn) [inline props/saber_BC_buttons.h:940]: How about "random enabled" / "random disabled" ?

Comment (NoSloppy) [reply props/saber_BC_buttons.h:940]: Love that, thanks.

Comment (profezzorn) [reply props/saber_BC_buttons.h:938]: You can move the SayRandom() to before the if statement since it is the same for both branches.

Comment (NoSloppy) [reply props/saber_BC_buttons.h]: Yes, thank you.

**Pull request merged**